blazar_dashboard/api/client.py

```diff
@@ -341,70 +355,10 @@ def network_reservation_dict(reservation, resource_id):
     return networks, list(chain(*network_reservations))
 
 
-def extra_capability_names(request):
-    """
-    Return all the names for possible selections.
-    """
-    cursor = get_cursor_for_request(request)
-    sql = '''\
-    SELECT DISTINCT
-        capability_name
-    FROM
-        computehost_extra_capabilities
-    '''
-    cursor.execute(sql)
-    # available = dictfetchall(cursor)
-    available = [row[0] for row in cursor.fetchall()]
-
-    cursor = get_cursor_for_request(request)
-    sql = '''\
-    SELECT DISTINCT
-        capability_name
-    FROM
-        networksegment_extra_capabilities
-    '''
-    cursor.execute(sql)
-    # available = dictfetchall(cursor)
-    available += [row[0] for row in cursor.fetchall()]
-    return list(set(available))
-
-
-def extra_capability_values(request, name):
-    """
-    Return the capabilities with a given "name". The client can cache/combine
-    the rows together to build up a full copy of the extra capabilities table
-    if they really want. Then they can do the "what-if" filtering themselves
-    to count hosts.
-
-    They could maybe mix that with the calendar data to even see if the chosen
-    number of hosts are free. Might need to do the lookup between
-    computehost_id (small integers) and the UUID via the uid name:value pairs.
-    """
-    cursor = get_cursor_for_request(request)
```

Comment:
Is this `get_cursor_for_request` function still used anywhere? I hope not... if it's unused, let's delete it (it should fail lint, if proper linting is set up in the repo.)